### PR TITLE
Match number of ending message `<` with starting message `>` & color these messages yellow

### PR DESCRIPTION
**Pull request by itscaro**
## Motivation and Context

- [x] I have raised an issue to propose this change (#286)

## How Has This Been Tested?

image

## Types of changes

- [x] UX enhancement
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: commands/build.go [hunk 1]
```diff
@@ -113,16 +114,15 @@ func build(services *stack.Services, queueDepth int, shrinkwrap bool) {
...
 				if len(function.Language) == 0 {
 					fmt.Println("Please provide a valid --lang or 'Dockerfile' for your function.")
-
 				} else {
 					builder.BuildImage(function.Image, function.Handler, function.Name, function.Language, nocache, squash, shrinkwrap)
 				}
+				fmt.Printf(aec.YellowF.Apply("[%d] < Building %s done.\n"), index, function.Name)
 			}
 
-			fmt.Printf("[%d] < Builder done.\n", index)
```
Comment (alexellis) [inline commands/build.go]: Can we keep this and maybe say: Builder [%d] finished.

File: commands/push.go [hunk 2]
```diff
@@ -71,15 +72,15 @@ func pushStack(services *stack.Services, queueDepth int) {
...
+				fmt.Printf(aec.YellowF.Apply("[%d] > Pushing %s.\n"), index, function.Name)
 				if len(function.Image) == 0 {
 					fmt.Println("Please provide a valid Image value in the YAML file.")
 				} else {
 					pushImage(function.Image)
 				}
+				fmt.Printf(aec.YellowF.Apply("[%d] < Pushing %s done.\n"), index, function.Name)
 			}
 
-			fmt.Printf("[%d] < Pushing done.\n", index)
```
Comment (alexellis) [inline commands/push.go]: Retain the message please as "Push worker [%d] done"

**Comment by itscaro** [comment]
@alexellis cf. the screenshot

**Comment by itscaro** [comment]
@alexellis I made the changes

File: commands/build.go (hunk 1 above)
Comment (itscaro) [reply commands/build.go]: `fmt.Printf(aec.YellowF.Apply("[%d] Builder finished.\n"), index)`
is it's ok with color and index as prefix ?

File: commands/push.go (hunk 2 above)
Comment (itscaro) [reply commands/push.go]: `fmt.Printf(aec.YellowF.Apply("[%d] Push worker done"), index)`
is it's ok with color and index as prefix ?

Comment (alexellis) [reply commands/push.go]: Maybe let's rephrase to "[%d] worker done." for both. Yellow is fine or normal.

**Comment by itscaro** [comment]
@alexellis Done

**Comment by alexellis** [comment]
Merging..

**Pull request merged**